.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+
+name: Bug Report
+about: Create a bug report to help us improve geopandas
+title: "BUG:"
+labels: "bug"
```

Comment:
I liked the initial version with `needs triage`, which indicates that reported bug has not been confirmed by the team (just requires adding the label).

Reply:
Okay, cool. We don't actually have that label in geopandas (which is why I removed it), but I'll add it back here and create that label.